### DISCUSSION TITLE
Enable the {{ markdown }} handlebars block helper

### DISCUSSION THREAD

**Discussion by tglynx** [Budibase/budibase#11198] [Ideas]
Hi,

now that we can edit markdown with the nice rich text form field it would be nice to reenable the {{ markdown }} block helper in handlebars-helpers. It has been disabled 2 years ago. 
I am not aware of the negative impact - or the reason why it has been disabled, but it would enable to send HTML formatted E-Mails that have been created using a markdown field (as an example) ...

One would just pass the content of the form field to an automation with a preconfigured email template like this: 

{{ #markdown }}
{{ Form.Fields.Message }}
{{ /markdown }}

The helper would convert the markdown to HTML and there we go....

Regards,
Michael

**Comment by melohagan** [comment]
Hey @tglynx 

It appears the markdown helper was removed for performance reasons in this PR here: https://github.com/Budibase/budibase/pull/4312

@aptkingston May be able to provide a little more context on this one, although it was quite a while ago now!
